### ISSUE TITLE
Ctrl+s discards edited article instead of saving it

### ISSUE THREAD

**Issue by makermann** [joomla/joomla-cms#16550]
### Steps to reproduce the issue
Create a new article or open an existing article.
Edit the article text. Or change an image etc.
Press the keys Ctrl and S (Ctrl+s).

### Expected result
The edited version of the article is saved. Or no action, so the changed text ist still available.

### Actual result
The page is reloaded. The user is directed to the article list page (com_content, view=articles). The article is checked out. All changes are lost.

### System information (as much as possible)
Server: Joomla 3.7.2., php 7, Using default tinymce as the editor plugin, standard out of the box Joomla, no plugins for article editing, no K2.

Desktop: Suse Linux, Browser: Firefox. 

### Additional comments
I have lost multiple edits due to the engrained automatic pressing of Ctrl+s.
Hence the behavior of the Joomla article editor-view has led me to lose hours of my valuable time. So right now the only solution is, not to use the article editor for writing articles (only for entry). Which is the purpose of the editor in the first place.

**Comment by franz-wohlkoenig** [comment]
Using cmd+s, alt+s or ctrl+s doesn't reload Page. cmd+s open the Dialog to save as html-Site.
### System information
3.7.3-dev (latest nightly build)
Multilanguage Site
macOS Sierra, 10.12.5
Firefox 53 (64-bit)
#### MAMP 4.1.1
- PHP 7.0.15
- MySQLi 5.6.35

**Comment by AlexRed** [comment]
I can confirm:
Edit the article text.
Press the keys Ctrl and S (Ctrl+s).
The page is reloaded. The user is directed to the article list page (com_content, view=articles). The article is checked out. All changes are lost.

Test in Joomla 3.7.3Dev  from Firefox on Linux Ubuntu<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16550">issues.joomla.org/tracker/joomla-cms/16550</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
3x Firefox, but Issue only on Linux. Can someone test on Windows?

**Comment by brianteeman** [comment]
**tested chrome on windows**
Ctrl-s will close the editor and return to the list view without saving
**tested edge on windows**
Ctrl-s will close the editor and return to the list view without saving
**tested ie11 on windows**
Ctrl-s will close the editor and return to the list view without saving
**tested firefox on windows**
Ctrl-s will close the editor and return to the list view without saving

## NOTE to testers
This only happens when the cursor is in the tinymce editor - if it is anywhere else on the page then cyrl-s will open a browser dialogue to try and save the **_page_** locally

**Comment by franz-wohlkoenig** [comment]
@brianteeman you're right: also on Mac using cmd+s and Curser is in TinyMCE-Editor the Page gets reloaded and Article checked-out in List-View without saving Changes.

**Comment by brianteeman** [comment]
Related info https://www.tinymce.com/docs/plugins/save/

**Comment by dgt41** [comment]
Few months ago we had that code in J4. now that code is not there anymore... 😞
![screen shot 2017-06-09 at 20 48 50](https://user-images.githubusercontent.com/3889375/26987664-1c79873e-4d55-11e7-8508-5b2ebc09776e.png)

**Comment by brianteeman** [comment]
It's not related to that. I gave the tinymce link that is the cause earlier

**Comment by dgt41** [comment]
@brianteeman following the instructions of the link you pasted above, this can be fixed by adding this:
```js
		if (tinyMCE && typeof tinyMCE === 'function') {
			button.type = 'submitbtn';
		}
```
here: https://github.com/joomla/joomla-cms/blob/staging/media/system/js/core-uncompressed.js#L70

**Comment by ghazal** [comment]
@dgt41 I tried your solution. It doesn't work (mac here). Result is the same as described above.

**Comment by wojsmol** [comment]
@ghazal Have you also made changes to https://github.com/joomla/joomla-cms/blob/staging/media/system/js/core.js. If not, the changes will be visible only when system debug mode is enabled

**Comment by ghazal** [comment]
Yes, of course ...

**Comment by makermann** [comment]
Hi, thanks for taking the time to investigate the problem. Has there been any progress to the issue?

**Comment by franz-wohlkoenig** [comment]
@makermann there is no PR solving this Issue. I understand @brianteeman 's [comment](https://github.com/joomla/joomla-cms/issues/16550#issuecomment-306738748) as its an Issue at TinyMCE – correct, brian?

**Comment by brianteeman** [comment]
I have worked out how to prevent ctrl-s from closing the window and only offering a browser dialog to save. This is by disabling the save plugin in tinymce.php

That will prevent the original report of losing an article but it doesnt make ctrl-s save the article.

**Comment by brianteeman** [comment]
I can no longer replicate the issue. I am going to assume it has been resolved elsewhere but if not and someone can replicate it then it can always be reopened

**Comment by ghost** [comment]
Reopened as requested in https://github.com/joomla/joomla-cms/issues/26197#issuecomment-529105004

**Comment by Quy** [comment]
Please test PR #27519
